NDB.Covid19/NDB.Covid19/ViewModels/InformationAndConsentViewModel.cs

```diff
@@ -90,6 +102,7 @@ void OnAuthCompleted(object sender, AuthenticatorCompletedEventArgs e)
             if ((e?.IsAuthenticated ?? false) && e.Account?.Properties != null && e.Account.Properties.ContainsKey("access_token"))
             {
                 LogUtils.LogMessage(Enums.LogSeverity.INFO, errorMsgPrefix + "User returned from ID Porten after authentication and access_token exists.");
+                AreCountryConsentsGiven = false;
```

Comment:
Is it also reset in the preferences as well?

Reply:
Yes